Library/Homebrew/keg_relocate.rb

```diff
@@ -2,50 +2,12 @@ class Keg
   PREFIX_PLACEHOLDER = "@@HOMEBREW_PREFIX@@".freeze
   CELLAR_PLACEHOLDER = "@@HOMEBREW_CELLAR@@".freeze
 
-  def fix_install_names
-    mach_o_files.each do |file|
-      file.ensure_writable do
-        change_dylib_id(dylib_id_for(file), file) if file.dylib?
-
-        each_install_name_for(file) do |bad_name|
-          # Don't fix absolute paths unless they are rooted in the build directory
-          next if bad_name.start_with?("/") && !bad_name.start_with?(HOMEBREW_TEMP.to_s)
-
-          new_name = fixed_name(file, bad_name)
-          change_install_name(bad_name, new_name, file) unless new_name == bad_name
-        end
-      end
-    end
-
-    symlink_files.each do |file|
-      link = file.readlink
-      # Don't fix relative symlinks
-      next unless link.absolute?
-      if link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
-        FileUtils.ln_sf(link.relative_path_from(file.parent), file)
-      end
-    end
+  def fix_install_linkage
+    []
   end
 
-  def relocate_install_names(old_prefix, new_prefix, old_cellar, new_cellar)
-    mach_o_files.each do |file|
-      file.ensure_writable do
-        if file.dylib?
-          id = dylib_id_for(file).sub(old_prefix, new_prefix)
-          change_dylib_id(id, file)
-        end
-
-        each_install_name_for(file) do |old_name|
-          if old_name.start_with? old_cellar
-            new_name = old_name.sub(old_cellar, new_cellar)
-          elsif old_name.start_with? old_prefix
-            new_name = old_name.sub(old_prefix, new_prefix)
-          end
-
-          change_install_name(old_name, new_name, file) if new_name
-        end
-      end
-    end
+  def relocate_install_linkage(old_prefix, new_prefix, old_cellar, new_cellar)
```

Comment:
Let's call this `relocate_dynamic_linkage`. I think that's more understandable and better reflects the reality.